app/classifier/annotation-renderer/svg.jsx

```diff
@@ -189,7 +191,7 @@ export default class SVGRenderer extends React.Component {
                 />
               </Draggable>
             )}
-
+            <ProgressMarker />
```

Comment:
I'm not sure about adding the progress marker to every subject image. I think that's why the tests are failing.